connect_ferc1_to_eia.py

```diff
@@ -132,7 +132,9 @@ def prep_ferc_data(pudl_out):
                    'installation_year',
                    'primary_fuel_by_mmbtu',
                    'plant_type',
-                   'record_id']
+                   'record_id',
+                   'opex_production_total',
+                   ]
```

Comment:
This was not necessary for connecting ferc to eia, but I added `opex_production_total` in here bc that is the main data column I actually want from the steam table.